Source/Concurrency/TransitionRelationComputation.cs

```diff
@@ -370,7 +370,7 @@ private void EliminateIntermediateVariables()
       {
         TryElimination(Enumerable.Empty<Variable>());
         TryElimination(trc.allLocVars.Select(v => varCopies[v][0]));
-        TryElimination(trc.allLocVars.Where(v => varCopies[v].Count > 1).Select(v => varCopies[v][1]));
```

Comment:
Why not have both?

Reply:
We could have both. It would be great to have a comment right there indicating all the possibilities that exist. 

I chose to minimize the number of different attempts because that seemed like the principled thing to do. I even tried to remove the attempt on line 372 but some of the tests started failing.